web/src/beta/features/Navbar/Menus/ProjectMenu/index.tsx

```diff
@@ -16,24 +16,28 @@ const ProjectMenu: React.FC<Props> = ({ currentProject, workspaceId }) => {
   const menuItems: MenuItem[] = [
     {
       text: t("Project settings"),
-      linkTo: `/settings/projects/${currentProject.id}`,
+      linkTo: `/settings/project/${currentProject.id}`,
     },
     {
-      text: t("Datasets"),
-      linkTo: `/settings/projects/${currentProject.id}/dataset`,
+      text: t("Story"),
+      linkTo: `/settings/project/${currentProject.id}/story`,
     },
     {
-      text: t("Plugins"),
-      linkTo: `/settings/projects/${currentProject.id}/plugins`,
+      text: t("Public"),
+      linkTo: `/settings/project/${currentProject.id}/public`,
     },
     {
-      text: t("Plugins"),
-      linkTo: `/settings/projects/${currentProject.id}/plugins`,
+      text: t("Workspace assets"),
+      linkTo: `/settings/project/${currentProject.id}/asset`,
```

Comment:
Did you talk with @lavalse about assets? Since they are suppose to be connected with a workspace, this seems very misleading...

Reply:
As discussed we may leave it here for now.